src/components/PostInfo/PostInfo.jsx

```diff
@@ -1 +1,26 @@
-export const PostInfo = () => <>Put the post here</>;
+import { CommentList } from '../CommentList/CommentList';
+import { UserInfo } from '../UserInfo/UserInfo';
+import './PostInfo.scss';
+
+export const PostInfo = ({ post }) => (
+  <div className="PostInfo">
+    <div className="PostInfo__header">
+      <h3 className="PostInfo__title">{post.title}</h3>
+
+      <p>
+        {' Posted by  '}
+        <UserInfo user={post.user} />
```

Comment:
```suggestion
       {post.user && <UserInfo user={post.user} />}
```
find can return null, so better to add the check